src/components/Main/main.container.js

```diff
@@ -10,9 +10,19 @@ export const MainContainer = observer(({navigation}) => {
   const translateY = useSharedValue(-200);
 
   useEffect(() => {
-    AsyncStorageService.getBookmarks()
-      .then(bookmarkStore.setBookmarks)
-      .catch((e) => console.log('getBookmark error ', e));
+    (async () => {
+      try {
+        const bookmarks = await AsyncStorageService.getBookmarks();
+
+        bookmarks ?
+          bookmarkStore.setBookmarks(bookmarks)
+          :
+          bookmarkStore.setBookmarks([]);
```

Comment:
bookmarkStore.setBookmarks(bookmarks ?? []) (maybe || instead of ??, check out)

Reply:
don't use ternary operators for conditional function invokation, use it only for conditional assignment